test/unit/kubernetes-deploy/runner_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'test_helper'
+
+class RunnerTest < Minitest::Test
+  def setup
+    @logger_stream = StringIO.new
+    @logger = Logger.new(@logger_stream)
+    KubernetesDeploy.logger = @logger
+  end
+
+  def teardown
+    @logger_stream.close
+  end
+
+  def test_that_it_has_a_version_number
+    refute_nil ::KubernetesDeploy::VERSION
+  end
+
+  def test_initializer
+    # TODO: Switch to --kubeconfig for kubectl shell out and pass env var as arg to Runner init
+    # Then fix this crappy env manipulation
+    original_env = ENV["KUBECONFIG"]
+    ENV["KUBECONFIG"] = "/this-really-should/not-exist"
```

Comment:
I'm not exactly sure I understand your comment but it sounds like you're changing this anyway. A helper for this would be good  `with_kube_config` or `with_env(kubeconfig: path)` , but it sounds like you're planning to do something like this anyway.